zip-0316.html

```diff
@@ -174,6 +174,7 @@
                     <li><em>prefix</em> || “<code>view</code>” for Unified Full Viewing Keys on Mainnet;</li>
                     <li><em>prefix</em> || “<code>viewtest</code>” for Unified Full Viewing Keys on Testnet.</li>
                 </ul>
+                <p>While support for Revision 1 UAs/UVKs is still being rolled out across the Zcash ecosystem, a Producer can maximize interoperability by generating a Revision 0 UA/UVK in cases where the conditions on its use are met (i.e. there are no MUST-understand Metadata Items, and at least one shielded item). At some point when Revision 1 UA/UVKs are widely supported, this will be unnecessary and it will be sufficient to always produce Revision 1 UA/UVKs.</p>
```

Comment:
Note from Daira-Emma: remove the rendering from the PR, now that we have rendering automation.